third_party/move/move-model/src/ast.rs

```diff
@@ -1007,6 +1007,30 @@ impl ExpData {
         }
     }
 
+    /// A function which can be used by a `node_rewriter` argument to `ExpData::rewrite_node_id` to
+    /// update node location (`Loc`), in addition to instantiating types.
```

Comment:
Maybe also add some guidance on when to use which method (this one vs. the one above)?

Reply:
Added something.